arch/cortex-m/src/scb.rs

```diff
@@ -54,3 +54,8 @@ pub unsafe fn reset() {
     let reset = (0x5FA << 16) | (aircr & (0x7 << 8)) | (1 << 2);
     SCB.aircr.set(reset);
 }
+
+/// relocate interrupt vector table
+pub unsafe fn set_vector_table_offset(offset: u32) {
```

Comment:
`offset` is really an address, correct? Should the argument be a pointer?

Reply:
VTOR is [Vector Table Offset Register](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dai0321a/BIHHDGBC.html)

Not sure if the `offset` is an address or not

Reply:
ARM specifies it as an offset, but that offset appears to always be from 0. I think it is more clear of an API if we take a pointer as an argument, which is the location the vector table is now at.

Reply:
Still new to Rust, not sure what  pointer type should be used? `*const c_void` or `*const u32`?

Reply:
probably `*const fn()` since the VTOR is mostly composed of function pointers, or `*const ()`.